client/static/app.js

```diff
@@ -0,0 +1,86 @@
+const searchbutton = document.querySelector('#searchbutton');
+const form = document.querySelector('form');
+const searchbar = document.querySelector('#searchbar');
+const lucky = document.getElementById('lucky');
+const resultsArea = document.querySelector('#results');
+// const randLink = document.getElementById('randLink');
```

Comment:
Any reasons for sometimes using querySelector and sometimes getElementById (for my own understanding)?